ecosystem/resources/members/OMM.toml

```
@@ -0,0 +1,10 @@
+name = "qOMM"
+url = "https://github.com/renlililoli/OMM"
+description = "The updated qOMM, which now can run with qiskit==1.1.0 version. It is an implementation of the algorithm from the paper https://arxiv.org/abs/2201.07963."
```

Thanks! At the moment, this doesn't tell readers much about what the package is. Can I suggest something like the following?

```suggestion
description = "Implementation of the quantum orbital minimization method to calculate excited states."
```

---

Sure :).